conans/cli/commands/search.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from conans.client.output import Color
+from conans.errors import ConanException
+from conans.cli.command import OnceArgument, Extender, conan_command
+
+
+def output_search_cli(info, out):
+    results = info["results"]
+    for remote_info in results:
+        source = "cache" if remote_info["remote"] is None else str(remote_info["remote"])
+        out.writeln("{}:".format(source), Color.BRIGHT_WHITE)
+        for conan_item in remote_info["items"]:
+            reference = conan_item["recipe"]["id"]
+            out.writeln(" {}".format(reference))
+
+
+def output_search_json(info, out):
+    myjson = json.dumps(info["results"], indent=4)
+    out.writeln(myjson)
+
+
+@conan_command(group="Consumer commands", cli=output_search_cli, json=output_search_json)
```

Comment:
I am not convinced of the approach of taking extra kwargs as formatters, I think this can bite us in the future. I would try to be explicit: ``formatters={"cli": output_search_cli, "json": output_search_json}``

Reply:
Not possible to not use kwargs for cli, json formatters?

Reply:
https://github.com/conan-io/conan/blob/f12f38c39877549f4a0d1da4830bd0ed77de61a3/conans/cli/command.py#L24-L28

Reply:
I have changed it to pass a dictionary as the parameter of the decorator